glass/fields.py

```diff
@@ -608,3 +643,137 @@ def effective_cls(
         if weights2 is weights1 and j1 != j2:
             out[j2 + j1] = cl
     return out
+
+
+def gaussian_fields(shells: Sequence[RadialWindow]) -> Sequence[grf.Normal]:
+    """Create Gaussian random fields for radial windows *shells*."""
```

Comment:
Explicit "Parameters" and "Returns" missing in docstring

Reply:
I am really concerned with a lot of these docstrings. I put in a lot of work to make the docs in a really good place. By leaving all these as one-liners we are essentially undoing a massive part of work.

Reply:
I don't think this is taking anything away from the existing documentation. At the moment, users seem to struggle with the rationale of how things work, so my hope is that this will improve with more of a narrative documentation than just API reference (think Python docs vs NumPy docs). But I don't have the bandwidth to fully write this up now. Happy to add Parameters and Returns to these.